<ui version="4.0" >
 <author>José Manuel Díez Botella</author>
 <class>buscaconcepto</class>
 <widget class="QDialog" name="buscaconcepto" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Búsqueda de conceptos</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="TextLabel1" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text" >
          <string>CLAVE:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="claveLineEdit" >
         <property name="maximumSize" >
          <size>
           <width>80</width>
           <height>32767</height>
          </size>
         </property>
         <property name="maxLength" >
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="TextLabel2" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text" >
          <string>DESCRIPCIÓN:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="descripcionLineEdit" >
         <property name="maxLength" >
          <number>80</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Selección</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>9</number>
        </property>
        <property name="topMargin" >
         <number>9</number>
        </property>
        <property name="rightMargin" >
         <number>9</number>
        </property>
        <property name="bottomMargin" >
         <number>9</number>
        </property>
        <item>
         <widget class="QRadioButton" name="ComienceRadioButton" >
          <property name="text" >
           <string>Que comience por ...</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ContengaRadioButton" >
          <property name="text" >
           <string>Que contenga ...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="AceptarBoton" >
       <property name="minimumSize" >
        <size>
         <width>120</width>
         <height>35</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <family>Helvetica [Adobe]</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>&amp;Aceptar</string>
       </property>
       <property name="shortcut" >
        <string>Alt+A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelarBoton" >
       <property name="minimumSize" >
        <size>
         <width>120</width>
         <height>35</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <family>Helvetica [Adobe]</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>&amp;Cancelar</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView" />
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <includes>
  <include location="local" >funciones.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>AceptarBoton</sender>
   <signal>clicked()</signal>
   <receiver>buscaconcepto</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>455</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel" >
     <x>402</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelarBoton</sender>
   <signal>clicked()</signal>
   <receiver>buscaconcepto</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>618</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel" >
     <x>618</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
